data/net/minecraft/resources/ResourceKey.mapping

```diff
@@ -1,14 +1,45 @@
 CLASS net/minecraft/resources/ResourceKey
+	COMMENT A class representing an immutable key for a resource, in terms of the name of its
```

Comment:
Small bit of pedantry (ignore at your discretion): The first part of the sentence can be shortened to `An immutable key for a resource,`, since the name of the class and its definition makes it clear that it is a class.